content/videos/challenges/104-linear-regression-with-tensorflowjs/index.json

```
@@ -0,0 +1,96 @@
+{
+    "title": "Linear Regression with TensorFlow.js",
+    "description": "In this challenge, I use the TensorFlow.js library to create an interactive demonstration of linear regression (with stochastic gradient descent!)",
+    "videoNumber": "104",
+    "videoId": "dLp10CFIvxI",
+    "date": "2018-05-29",
+    "languages": ["p5.js", "tensorflow.js"],
+    "topics": [
+        "tensorflow js",
+        "linear regression",
+        "stochastic gradient descent"
+    ],
+    "canContribute": true,
+    "relatedChallenges": [],
+    "timestamps": [
+        { "time": "0:00", "title": "Introduction" },
+        { "time": "1:21", "title": "What is linear regression?" },
+        { "time": "3:16", "title": "What do we need?" },
+        { "time": "3:41", "title": "Loss function" },
+        { "time": "4:48", "title": "Optimizer" },
+        { "time": "5:26", "title": "Formula for line" },
+        { "time": "8:02", "title": "Let's Code!" },
+        { "time": "8:23", "title": "Add data set -- mouse clicks" },
+        { "time": "9:16", "title": "Remap x, y to 0, 1" },
+        { "time": "12:12", "title": "Randomly initialize m and b" },
+        { "time": "13:10", "title": "tf.variable()" },
+        { "time": "13:39", "title": "predict()" },
+        { "time": "16:13", "title": "Optimizer & learning rate" },
+        { "time": "17:25", "title": "Loss function" },
+        { "time": "19:24", "title": "Minimize loss" },
+        { "time": "22:53", "title": "What is the optimizer doing?" },
+        { "time": "27:42", "title": "Visualize regression results" },
+        { "time": "32.49", "title": "dataSync()" },
+        { "time": "35:00", "title": "Adjust learning rate" },
+        { "time": "35:35", "title": "Memory management" },
+        { "time": "41:01", "title": "Improvements" }
+    ],
+    "codeExamples": [
+        {
+            "title": "Linear Regression",
+            "description": "This sketch creates an interactive simulation of linear regresssion.",
+            "image": "",
+            "urls": {
+                "p5": "https://editor.p5js.org/codingtrain/sketches/UtOWCSYYF"
+            }
+        }
+    ],
+    "groupLinks": [
+        {
+            "title": "References",
+            "links": [
+                {
+                    "icon": "💾",
+                    "title": "TensorFlow.js",
+                    "url": "https://js.tensorflow.org/",
+                    "description": "Webpage with reference materials and documentation about TensorFow.js"
+                },
+                {
+                    "icon": "📈",
```

```suggestion
                    "icon": "🐦",
```

We could use a bird!